catroid/build.gradle

```diff
@@ -59,6 +60,7 @@ android {
             buildConfigField "boolean", "FEATURE_ARDUINO_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_RASPI_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_POCKETMUSIC_ENABLED", "true"
+            buildConfigField "boolean", "FIREBASE_CRASH_REPORT", "false"
```

Comment:
In the spirit of consistency you should consider naming it `FIREBASE_CRASH_REPORT_ENABLED`. 😼